src/main/resources/io/jenkins/plugins/designlibrary/Select/index.properties

```diff
@@ -1,7 +1,8 @@
-title=There are a couple of ways to create a dropdown in Jenkins.
 usage.1=You can use <code>dropdownDescriptorSelector</code> to create a fully data-bound form and the option to include a \
 config page for the <code>descriptor</code> by creating a file called <code>config.jelly</code>.
 usage.2=If you want a simpler <code>select</code> to be rendered you can use the <code>f:select</code> \
 Jelly tag and create a <code>doFillFieldNameItems</code> method in the <code>descriptor</code>.
 dynamic=Dynamic select
 dynamic.description=Updates the contents of a "select" control dynamically based on selections of other controls.
+searchable=Searchable select
```

Comment:
I don't think this is actually a searchable select, its an auto complete, but I don't think you are forced to use the text from this (would need to check)